src/components/Recipe/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { createPortal } from 'react-dom'
+
+import { fetchRecipe } from 'api'
+import Loader from 'components/common/Loader'
+import { Ingredient, RecipeDetails, Step } from 'types'
+
+import styles from './Recipe.module.scss'
+
+function Recipe() {
+  const { id } = useParams()
+
+  const [recipe, setRecipe] = useState<RecipeDetails>()
+  const [isLoading, setIsLoading] = useState<boolean>(true)
+
+  useEffect(() => {
+    fetchRecipe(id!).then((response) => {
+      setRecipe(response.data)
+      setIsLoading(false)
+    })
+  }, [id])
```

Comment:
isn't id just recipeName?

Reply:
@sasza2 id is a number. API endpoint looks like this `/recipes/469824/information`